playground/kwj1270/commerce/infra/k6/performance.sh

```
@@ -0,0 +1 @@
+k6 run --out influxdb=http://localhost:8086/k6 display.js
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)